iceoryx2/src/config.rs

```diff
@@ -69,22 +69,20 @@
 //! # }
 //! ```
 
+use dirs;
 use iceoryx2_bb_container::semantic_string::SemanticString;
 use iceoryx2_bb_elementary::lazy_singleton::*;
 use iceoryx2_bb_posix::{file::FileBuilder, shared_memory::AccessMode};
 use iceoryx2_bb_system_types::file_name::FileName;
 use iceoryx2_bb_system_types::file_path::FilePath;
 use iceoryx2_bb_system_types::path::Path;
 use serde::{Deserialize, Serialize};
-use std::time::Duration;
+use std::{os::unix::ffi::OsStrExt, time::Duration};
```

Comment:
This will not work on Windows (unix-only).

Reply:
Hi @orecham I fixed my code by adding a new crate abastraction layer to get the default directory of the configuration file. And I've added a new sub-command: now I can display the system configuration and show the current iceoryx2 configuration.